plugins/reach/lib/api/data/vendorCatalogItem/KalturaVendorLiveTranslationCatalogItem.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ */
+class KalturaVendorLiveTranslationCatalogItem extends KalturaVendorLiveCatalogItem
+{
+	/**
+	 * @var KalturaCatalogItemLanguage
+	 * @filter eq,in
+	 */
+	public $targetLanguage;
+
+	private static $map_between_objects = array
+	(
+		'targetLanguage'
+	);
+
+	public function getMapBetweenObjects()
+	{
+		return array_merge(parent::getMapBetweenObjects(), self::$map_between_objects);
+	}
+
+	protected function getServiceFeature()
+	{
+		return KalturaVendorServiceFeature::LIVE_TRANSLATION;
+	}
+
+	protected function validateTargetLanguage(VendorCatalogItem $sourceObject = null)
+	{
+		if ($this->targetLanguage == 'Auto Detect')
```

Comment:
why not using the enum from plugins/reach/lib/api/enums/KalturaCatalogItemLanguage.php:73?